### DISCUSSION TITLE
Quale allarme filare nuovo, integrabile in home assistant

### DISCUSSION THREAD

**Discussion by nrizzano** [indomus/forum#2419] [HUB: Home Assistant]
Buongiorno,
ho acquistato casa nuova, vorrei sostituire l'attuale centrale allarme AXEL Atlantis64 con una già compatibile con Home Assistant.
Nel vecchio appartamento avevo una vecchia centrale Lince a 5 uscite che ho integrato personalmente tramite ESP32 ed alcuni relè connessi alla morsettiera e che comunicano lo stato e consentono di attivare/disattivare le varie uscite, poi tramite MQTT il mio ESP32 invia ad Home assistant i segnali per il pannello di allarme.
La centrale Axel in casa nuova non ha una morsettiera "analogica" ed è più complessa da integrare (non ho trovato documentazione e non posso, per mancanza di tempo, studiarmela a dovere).
A questo punto preferirei averne una nuova, magari da integrare tramite protocollo standard (http, rest, mqtt), qualcosa che mi consenta di mantenere l'allarme filare con tutti i sensori già cablati tra porte, finestre, sensori movimento, barriere, anti sfondamento... c'è già tutto e vorrei mantenere tutto.

Purtroppo non riesco a trovare nessuna centrale che sia "aperta" a comunicare con i sistemi esterni, tutti fanno una domotica base e ti impongono di usare il loro sistema, ma io vorrei poter usare HomeAssistant.
Alla fine quello che deve fare la centrale è comunicarmi lo stato (allarmata SI/NO, in allarme SI/NO, uscite aperte/chiuse (magari singolarmente)), e consentire l'attivazione/disattivazione da remoto (arm_night/arm_away), poi mi configuro io tutti gli scenari possibili e le notifiche varie direttamente da HA.
Su web trovo solo centrali con sensori wireless e le case costruttrici, interpellate, non rispondono.
Qualche suggerimento?

Grazie

**Comment by Maspar79** [comment]
Buonasera, hai provato a dare un occhiata a questo : https://konnected.io/

**Reply by ortorob73** [reply]
Ciao

ho provato ma non hanno le caratteristiche indicate?

Grazie

**Comment by nrizzano** [comment]
Ciao, da quello che vedo, Konnected è la versione commerciale (e fatta bene) di quello che ho fatto io (più o meno bene, ma funziona e tanto mi basta) sulla mia vecchia centrale.
Entrambe le soluzioni presuppongono che la centrale da interfacciare abbia delle morsettiere che consentano di leggere dei segnali in bassa tensione per capire se l'allarme è scattato, se la centrale è allarmata, ed altre morsettiere con contatti impulsivi che abilitano/disabilitano le zone dell'allarme.
La mia vecchia centrale ha queste morsettiere ed è documentata. La centrale esistente della nuova casa, non ha tutti questi ingressi, non forniscono documentazione e non saprei come collegarmici.
Probabilmente si dialoga in formato digitale con un protocollo proprietario che ovviamente non conosco (e la casa si guarda bene dal divulgarne le informazioni).
Quindi Konnected, purtroppo, non va bene nel mio caso.
Non mi interessa più di tanto recuperare la vecchia centrale, infatti vorrei acquistarne una nuova che supporti tutti i sensori attualmente presenti dell'allarme filare, l'importante è che sia interfacciabile attraverso Home Assistant.
Mi sembra assurdo, ma non ne trovo!

**Reply by Maspar79** [reply]
Ciao, non installo normalmente centrali axel ma posso assicurarti che la tua atlantis non ha nessun "protocollo proprietario" se i tuoi sensori sono filari al massimo possono essere collegati con resistenze per il bilanciamento delle zone questo potrebbe esserti utile : https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.gam-service.it/images/pdf/AXEATLANTIS32.pdf&ved=2ahUKEwivnbult4X-AhWtYPEDHbjfA1oQFnoECA0QAQ&usg=AOvVaw3aiCrIyEWPMubnEGvX22cC

Per il resto non saprei, sinceramente quasi tutte le centrali sono sistemi abbastanza chiusi, magari qualche brand mette a disposizione delle api per poter dialogare con sistemi esterni, potresti vedere la risco mi pare di aver visto qualche custom components tempo fa.

Buona giornata

**Reply by nrizzano** [reply]
Ciao, non avrei problemi a sbarazzarmi della centrale Axel, se conosci altre centrali per le quali mi puoi garantire la compatibilità con home assistant, anche se ci devo smanettare un pochino, la prenderei al volo.
L'importante è che riesca a recuperare tutti i sensori che ho sparso per casa (credo di sì, dovrebbero essere dei semplici contatti puliti).

Intanto ti ringrazio.

**Comment by RiodaITA** [comment]
Ciao a tutti, io ho una centrale Jablotron serie 100 e l'ho interfacciata via USB a HA usando il modulo "jablotron 100+" presente in HACS. 
Il progetto lo trovate qua: https://github.com/kukulich/home-assistant-jablotron100
Il limite è che devi avere la centrale e l'hub di Home Assistant a una distanza ragionevole per un cavo USB, per il resto funziona tutto nel migliore dei modi, riesco a vedere lo stato di tutti i sensori (compresa la percentuale di carica delle batterie), posso attivare e disattivare le zone, usare lo stato o le variazioni di stato dei sensori per pilotare automazioni.
Nella pagina del progetto trovate tutte le informazioni necessarie.
Non credo ci sia compatibilità tra la centrale Jablotron e sensori di altri produttori che andrebbero sostituiti. D'altra parte però ti ritrovi con un sistema completamente funzionante su HA senza grandi sforzi.
Spero di essere stato utile

**Reply by DigoGibin** [reply]
Ciao, io ho integrato in home assistant una centralina inim smartliving 1050 con scheda lan/g. 
Non ha un integrazione nativa ma ci si può comunicare in modbus o con comandi rest vedo lo stato di tutti i sensori collegate i log della centralina e posso attivare disattivare aree senza problemi

**Reply by DigoGibin** [reply]
Si ti interessa approfondire ti mando qualche immagine ancora grezza ma di quello che si vede in home assistant

**Reply by nolith** [reply]
@DigoGibin io sarei interessato a dare un'occhiata. Ti trovi ancora bene con questa soluzione?

**Reply by DigoGibin** [reply]
Ciao, io si mi trovo bene fa tutto quello che mi aspetto che faccia senza perdere un colpo ... domani se vuoi ti mando un paio di immagini delle entità che ho estrapolato ora purtroppo non sono davanti al PC

**Reply by Tr4um4z** [reply]
> Ciao, io ho integrato in home assistant una centralina inim smartliving 1050 con scheda lan/g. Non ha un integrazione nativa ma ci si può comunicare in modbus o con comandi rest vedo lo stato di tutti i sensori collegate i log della centralina e posso attivare disattivare aree senza problemi

Buongiorno , sapresti indicarmi esattamente quale scheda di rete hai installato?

la PrimeLAN da specifiche da modbus (ma non credo sia per la centrale in questione)
https://www.inim.biz/it/antintrusione-e-domotica/rete-ethernet-e-gestione-ip/primelan?ref=17

la SmartLan/G invece non è specificato modbus nelle caratteristiche
https://www.inim.biz/it/antintrusione-e-domotica/rete-ethernet-e-gestione-ip/smartlang?ref=17

Grazie Anticipatamente

**Reply by DigoGibin** [reply]
> > Ciao, io ho integrato in home assistant una centralina inim smartliving 1050 con scheda lan/g. Non ha un integrazione nativa ma ci si può comunicare in modbus o con comandi rest vedo lo stato di tutti i sensori collegate i log della centralina e posso attivare disattivare aree senza problemi
> 
> Buongiorno , sapresti indicarmi esattamente quale scheda di rete hai installato?
> 
> la PrimeLAN da specifiche da modbus (ma non credo sia per la centrale in questione) https://www.inim.biz/it/antintrusione-e-domotica/rete-ethernet-e-gestione-ip/primelan?ref=17
> 
> la SmartLan/G invece non è specificato modbus nelle caratteristiche https://www.inim.biz/it/antintrusione-e-domotica/rete-ethernet-e-gestione-ip/smartlang?ref=17
> 
> Grazie Anticipatamente

Ciao, io ho installato la scheda Smartlan/G su una centrale inim smartliving 1050

**Comment by Argebombo** [comment]
> Si ti interessa approfondire ti mando qualche immagine ancora grezza ma di quello che si vede in home assistant

Ciao @DigoGibin sarei interessato alla cosa.

La tua soluzione è documentata da qualche parte?

Grazie 1000 in anticipo

**Comment by DigoGibin** [comment]
Ciao per la parte riguardante la comunicazione modbus non c'è documentazione, per le varie chiamate rest ho preso spunto dal forum di home assistant
[https://community.home-assistant.io/t/inim-alarm/60354/4](url)
C'è anche presente qui sul forum una mia discussione dove vengo aiutato con la dichiarazione di vari sensori
[https://github.com/indomus/forum/discussions/1845](url)
ecco alcune immagini di quello che ho ottenuto
![screen inim 1](https://github.com/indomus/forum/assets/88708826/09f9aa57-67ce-4b16-bc22-ad566aa94248)
![screen inim 2](https://github.com/indomus/forum/assets/88708826/d6761d62-68ac-4e59-b9eb-1a1a616ba0c8)
![Senza titolo 3](https://github.com/indomus/forum/assets/88708826/048992f2-ba45-48ae-b5b4-4270abee65a3)
![Senza titolo 4](https://github.com/indomus/forum/assets/88708826/d72d7c82-da11-4e78-9f7e-564318949b70)
ciao

**Reply by Argebombo** [reply]
> Ciao per la parte riguardante la comunicazione modbus non c'è documentazione, per le varie chiamate rest ho preso spunto dal forum di home assistant
> [https://community.home-assistant.io/t/inim-alarm/60354/4](url)
> C'è anche presente qui sul forum una mia discussione dove vengo aiutato con la dichiarazione di vari sensori
> [https://github.com/indomus/forum/discussions/1845](url)
> ecco alcune immagini di quello che ho ottenuto
> ![screen inim 1](https://github.com/indomus/forum/assets/88708826/09f9aa57-67ce-4b16-bc22-ad566aa94248)
> ![screen inim 2](https://github.com/indomus/forum/assets/88708826/d6761d62-68ac-4e59-b9eb-1a1a616ba0c8)
> ![Senza titolo 3](https://github.com/indomus/forum/assets/88708826/048992f2-ba45-48ae-b5b4-4270abee65a3)
> ![Senza titolo 4](https://github.com/indomus/forum/assets/88708826/d72d7c82-da11-4e78-9f7e-564318949b70)
> ciao

Ok grazie 1000

**Comment by blakdog1978** [comment]
Buongiorno,
sarei interessato a questa soluzione, non riesco ad aprire i link che hai postato, ma come comunichi con la centrale ?
Io ora sto usando addon creato da maxill ma ho qualche problema sulla pubblicazione dei nomi.

**Comment by DigoGibin** [comment]
Ciao non ho trovato nessun addon compatibile con le centrali smartliving, come scritto in precedenza uso comunicazione modbus tramite cavo di rete ma va attivata da un utente installatore nella centrale inim . un alternativa oppure un integrazione ibrida come il mio caso uso anche piattaforma restful di home assistant per attivazioni di scenari.
Se hai bisogno di una mano scrivimi in privato che cerco di aiutarti. 
Per i link se non ti funzionano copia e incolla l'indirizzo cosi dovrebbero andare
Ciao

**Comment by M4v3r1cK87** [comment]
Ciao a tutti,
riapro questa discussione per segnalare che da qualche tempo è online la mia integrazione con i sistemi della Lince Italia.

Potete trovare tutto qui: https://github.com/M4v3r1cK87/lince-alarm-for-home-assistant
